spec/GraphQLOverHTTP.md

```diff
@@ -745,3 +745,44 @@ payload is `application/json` then the client MUST NOT rely on the body to be a
 well-formed _GraphQL response_ since the source of the response may not be the
 server but instead some intermediary such as API gateways, proxies, firewalls,
 etc.
+
+# Non-normative notes
+
+## Security
+
+In this specification, GET requests are not supported for mutations due to
+security concerns. GET requests expose variables to logging mechanisms and
+intermediaries due to the URL encoding of parameters, which can lead to
+sensitive data being inadvertently logged. Furthermore, GET requests are
+considered "simple requests" under CORS (Cross-Origin Resource Sharing), meaning
+they bypass preflight checks that add a layer of security.
+
+On the other hand, using `application/json` for request bodies mandates a CORS
```

Comment:
This is a very nice addition.

The term “simple request” is used but not defined and I can’t find the definition in any of the references either. How about defining it using the following quote from the fetch spec? I believe that’s the authoritative source:

> For [requests](https://fetch.spec.whatwg.org/#concept-request) that are more involved than what is possible with HTML’s [form](https://html.spec.whatwg.org/multipage/forms.html#the-form-element) element, a [CORS-preflight request](https://fetch.spec.whatwg.org/#cors-preflight-request) is performed, to ensure [request](https://fetch.spec.whatwg.org/#concept-request)’s [current URL](https://fetch.spec.whatwg.org/#concept-request-current-url) supports the [CORS protocol](https://fetch.spec.whatwg.org/#cors-protocol).

https://fetch.spec.whatwg.org/#http-cors-protocol